ton_client/client/src/tvm/check_transaction.rs

```diff
@@ -19,6 +19,7 @@ use std::convert::TryFrom;
 pub(crate) async fn check_transaction<F>(

```

Comment:
What is the responsibility of this function? Why it is named check_transaction, but returns Fees?
And what is the skip_check parameter in function named check?
Suggest to rename this function to `calc_transaction_fees`.

Reply:
Done